decode.go

```diff
@@ -191,12 +196,21 @@ func (p *parser) node(kind Kind, defaultTag, tag, value string) *Node {
 }
 
 func (p *parser) parseChild(parent *Node) *Node {
-	child := p.parse()
+	child := p.parse(false)
 	parent.Content = append(parent.Content, child)
 	return child
 }
 
 func (p *parser) document() *Node {
+	if !p.event.implicit && len(p.event.head_comment) > 0 {
+		// This comment belongs to **before** the document event
+		n := p.node(DocumentNode, "", "", "")
+		// (1, 1) is not fully correct, but close enough
```

Comment:
Let's please just drop the comment here. Reading this code is clear that the lines are made up, and that's fine because the line is for the DocumentNode, not for the comment. Since the document is empty, there's nothing clearly wrong with this.

Reply:
Fixed in 193064d01c66d9643f3e6b9dd4142a2a83db62b4.